src/libraries/System.Private.Xml.Linq/tests/Schema/ExtensionTests.cs

```diff
@@ -27,397 +27,397 @@ namespace CoreXml.Test.XLinq
+            object elementValue = "AlteringElementValue";
+
+            // Alter XML document to invalidate.
+            var newElement = new XElement(elementName, elementValue);
+            xmlDocument.Root.Add(newElement);
+
+            validationSucceded = true;
+            ExtensionsClass.Validate(xmlDocument, schemaSet,
+                new ValidationEventHandler(TestValidationHandler));
+            Assert.False(validationSucceded);
+        }
+
+        /*
+         * test if exception is thrown when xml document does not
+         * conform to the xml schema
+         */
+        [Fact]
+        public void XDocumentThrowExceptionValidate()
+        {
+            string elementName = "AlteringElementName";
+            object elementValue = "AlteringElementValue";
+
+            // Alter XML document to invalidate.
+            var newElement = new XElement(elementName, elementValue);
+            xmlDocument.Root.Add(newElement);
+
+            Assert.Throws<XmlSchemaValidationException>(() => ExtensionsClass.Validate(xmlDocument, schemaSet, null));
+        }
+
+        /*
+         * test xml validation populating the XML tree with
+         * the post-schema-validation infoset(PSVI)
+         */
+        [Fact]
+        public void XDocumentAddSchemaInfoValidate()
+        {
+            // No. of elements before validation.
+            IEnumerable<XElement> elements = xmlDocument.Elements();
+            IEnumerator<XElement> elementsEnumerator = elements.GetEnumerator();
+            int beforeNoOfElements = 0;
+            int beforeNoOfAttributes = 0;
+            while (elementsEnumerator.MoveNext())
+            {
+                    beforeNoOfElements++;
+                    if (!elementsEnumerator.Current.HasAttributes)
+                        continue;
+
+                        IEnumerable<XAttribute> attributes = elementsEnumerator.Current.Attributes();
+                        IEnumerator<XAttribute> attributesEnumerator = attributes.GetEnumerator();
+                        while (attributesEnumerator.MoveNext())
+                            beforeNoOfAttributes++;
+            }
+
+            // Populate XML with PSVI.
+            validationSucceded = true;
+            ExtensionsClass.Validate(xmlDocument, schemaSet,
+                new ValidationEventHandler(TestValidationHandler), true);
+            Assert.True(validationSucceded);
+
+            // No. of elements after validation.
+            elements = xmlDocument.Elements();
+            elementsEnumerator = elements.GetEnumerator();
+            int afterNoOfElements = 0;
+            int afterNoOfAttributes = 0;
+            while (elementsEnumerator.MoveNext())
+            {
+                afterNoOfElements++;
+                if (!elementsEnumerator.Current.HasAttributes)
+                    continue;
+
+                    IEnumerable<XAttribute> attributes = elementsEnumerator.Current.Attributes();
```

Comment:
This block now needs to be unindented properly now that you've removed the braces.

Same for the other cases below.